benchmarks/cmp.c

```diff
@@ -86,7 +86,7 @@ int main (void)
 	long long nrIterations = 100000000;
 	const char str1[] = "some string to be compared with/some\\/more with a long common part, and only a bit different";
 	char * str2 = elektraMalloc (sizeof (str1));
-	strcat (str2, str1);
+	strcpy (str2, str1);
```

Comment:
Because somehow it always worked ^^ But now that `elektraMalloc` is `static inline` gcc complains about this line. That's actually the only reason I noticed this